### DISCUSSION TITLE
Add ability to view edit history for comments

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#107476] [Mobile] [answered]
GitHub Mobile beta for Android

I'd like to be able to view edit history for comments (in issues, pull requests, and discussions).

**Comment by HenSquared** [comment, marked as answer]
Hi @jsoref,
Thanks for the feedback.

I've created an Issue for this internally to discuss with the team. If we have further questions or updates, we will let you know here. 

Thanks!
